manifests/opensearch-bundle.1.0.yml

```diff
@@ -0,0 +1,53 @@
+---
+manifest:
```

Comment:
Not critical for the first iteration of this, but we should standardize this with #111 which suggests schema-version: 1.0

I'm thinking the input manifest and the copy included in the bundle should follow the same schema with the input being a subset of a final manifest.

Reply:
Done